store/store_ttl_test.go

```diff
@@ -216,7 +216,7 @@ func TestStoreWatchExpireEmptyRefresh(t *testing.T) {
 	fc := newFakeClock()
 	s.clock = fc
 
-	var eidx uint64 = 1
+	var eidx uint64
```

Comment:
will this fail the test? since the orginal code wants eidx to be 1

Reply:
It's not used till line 229, so `eidx` 1 isn't used until it is set as 3.

Reply:
i see.